### PR TITLE
UI: Footer Changes and Develop Merge

### PR DESCRIPTION
**Pull request by raveracker**

### CODE REVIEW

File: src/components/footer-menu.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { Link } from 'gatsby'
+import { MenuLink } from '../containers/header'
+
+interface FooterMenuProps {
+  className?: string
+  menuLinks: MenuLink[]
+}
+
+const FooterMenu: FC<FooterMenuProps> = ({ className, menuLinks }) => (
+  <div className="flex gap-8 md:gap-12 justify-center items-center mt-10">
```
Comment (ulhas) [inline src/components/footer-menu.tsx:11]: remove the implicit `mt-10` from the component. it should be passed as `className` prop or added to a wrapping div.

**Review by ulhas — CHANGES_REQUESTED** [review]

File: src/components/footer-menu.tsx (hunk 1 above)
Comment (raveracker) [reply src/components/footer-menu.tsx]: Done, I did not notice that earlier

File: src/containers/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+]
+
+const Footer: FC<FooterProps> = ({ menuLinks }) => {
+  return (
+    <div className="bg-info-900">
+      <div className="container flex flex-col items-center gap-4 py-10">
+        <Logo height="32" width="130" className="h-8 w-[8.5rem]" />
+        <FooterMenu
+          menuLinks={menuLinks}
+          className="flex gap-8 md:gap-12 justify-center items-center mt-10"
```
Comment (ulhas) [inline src/containers/footer.tsx:42]: check slack comment

**Review by ulhas — CHANGES_REQUESTED** [review]

File: src/containers/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import SocialIcons from '../components/social-icons'
+import Logo from '../assets/images/logo-white.svg'
+import Github from '../assets/images/github.svg'
+import { MenuLink } from '../containers/header'
+
+export interface FooterProps {
+  menuLinks: MenuLink[]
+}
+
+const socialLinks = [
```
Comment (pallavisaraogi) [inline src/containers/footer.tsx:12]: move this to `src/shared/sample-data.ts`

Comment (raveracker) [reply src/containers/footer.tsx:12]: These are jsx components, cannot be moved in a .ts file, also there would be changes on the data side that needs to be done after all the sections are done. The data part would be entirely done in graphql

**Review by pallavisaraogi — APPROVED** [review]

**Pull request merged**